components/splitter/panes.md

```diff
@@ -6,59 +6,301 @@ slug: splitter-panes
 
-````CSHTML
-@*Configure the Splitter Panes*@
+>caption Configure Splitter Panes
 
-<TelerikSplitter Width="600px" Height="400px">
+````CSHTML
+<TelerikSplitter Width="600px" Height="300px">
     <SplitterPanes>
 
-        <SplitterPane Collapsible="true" Size="200px" Min="100px" Max="300px">
+        <SplitterPane Collapsible="true" @bind-Size="@PaneSize1" Min="100px" Max="300px">
             <h4>Left Pane</h4>
-            <div>Collapsible pane with initial size in px that can be resized between 100px and 200px.</div>
+            <div>Collapsible and scrollable pane with initial size in px. It can be resized between 100px and 300px.</div>
         </SplitterPane>
 
-        <SplitterPane Collapsible="true" Size="20%">
+        <SplitterPane Collapsible="true" @bind-Size="@PaneSize2">
             <h4>Middle Pane</h4>
-            <div>Collapsible pane with initial size in percent.</div>
+            <div>Collapsible pane with initial size in per cent.</div>
         </SplitterPane>
 
-        <SplitterPane Collapsible="false" >
+        <SplitterPane Collapsible="false" Class="k-scrollable">
             <h4>Right Pane</h4>
-            <span>Non-collapsible pane. No size set, it will take the remaining space of the component.</span>
+            <div style="height:150%">Non-collapsible and scrollable pane with no size. It will take up the remaining space in the component.</div>
         </SplitterPane>
 
-        <SplitterPane Visible="false" >
-            <h4>Not Visible Pane</h4>
-            <span>This pane is not displayed.</span>
+        <SplitterPane Visible="false">
+            <h4>Invisible Pane</h4>
+            <span>This pane will not render.</span>
         </SplitterPane>
-        
+
     </SplitterPanes>
 </TelerikSplitter>
+
+@code {
+    private string PaneSize1 { get; set; } = "200px";
+
+    private string PaneSize2 { get; set; } = "20%";
+}
 ````
 
+## Pane Scrolling
+
+To make a Splitter pane scrollable without using additional HTML markup or custom CSS styles, set the pane `Class` parameter to `k-scrollable`. See the example above.
+
+## Pane Dimensions
+
+The dimensions of a Splitter pane depend on:
+
+* The [pane `Size`](#pane-size) parameter
+* The [pane `Collapsible` and `Resizable`](#pane-collapsibility-and-resizability) parameters
+* The [Splitter `Width`, `Height`, and `Orientation`](#splitter-width-and-height) parameters
+
+The sections below provide more details and a [hands-on example](#example).
+
+### Pane Size
+
+The Splitter pane `Size` controls the pane width or height, depending on the [Splitter `Orientation`]({%slug splitter-orientation%}).
+
+There must be at least one `SplitterPane` without a `Size`. This pane will adjust automatically to occupy the remaining space, based on the other pane sizes.
+
+If the pane `Size` is greater than `Max`, the pane cannot be resized even if its `Resizable` parameter is set to `true`.
```

Comment:
That's weird. I wouldn't expect it to be expandable, but why can't you make it smaller?

Reply:
The `Size` can be larger than `Max` only if the developer sets it like this. The user can never achieve such a state on their own. We consider it an invalid configuration, so we don't invest dev effort to fix it.